ProjFS.Mac/PrjFSKext/VnodeUtilities.cpp

```diff
@@ -6,7 +6,7 @@
 
 extern "C" int mac_vnop_getxattr(struct vnode *, const char *, char *, size_t, size_t *);
 
-FsidInode Vnode_GetFsidAndInode(vnode_t vnode, vfs_context_t context, bool useLinkIDForInode)
+FsidInode Vnode_GetFsidAndInode(vnode_t vnode, vfs_context_t _Nonnull context, bool useLinkIDForInode)
```

Comment:
Was there ever a valid reason for this to be null?

Reply:
No, I'll try removing this change and see if something breaks, it's not required for the vnode cache.

Reply:
Here's the error I'm seeing without this change:

```
In file included from /Users/wilbaker/Repos/VFSForGit/src/ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp:15:
/Users/wilbaker/Repos/VFSForGit/src/ProjFS.Mac/PrjFSKext/VnodeUtilities.hpp:14:57: error: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Werror,-Wnullability-completeness]
FsidInode Vnode_GetFsidAndInode(vnode_t _Nonnull vnode, vfs_context_t context, bool useLinkIDForInode);
                                                        ^
/Users/wilbaker/Repos/VFSForGit/src/ProjFS.Mac/PrjFSKext/VnodeUtilities.hpp:14:57: note: insert '_Nullable' if the pointer may be null
FsidInode Vnode_GetFsidAndInode(vnode_t _Nonnull vnode, vfs_context_t context, bool useLinkIDForInode);
                                                        ^
                                                                      _Nullable
/Users/wilbaker/Repos/VFSForGit/src/ProjFS.Mac/PrjFSKext/VnodeUtilities.hpp:14:57: note: insert '_Nonnull' if the pointer should never be null
FsidInode Vnode_GetFsidAndInode(vnode_t _Nonnull vnode, vfs_context_t context, bool useLinkIDForInode);
```